app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,5 @@
+class IngredientsController < ApplicationController
+  def index
+    @ingredients = Ingredient.all.sort_alpha
```

Comment:
This is good! One tidbit; when you do `Ingredient.` anything, it will assume you are doing `.all` in between. So, this could just read `Ingredient.sort_alpha` and it would be doing it on "all". Yours isn't wrong, just want to show you this alternative.